iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisConsensus.java

```diff
@@ -578,15 +597,26 @@ public void resetPeerList(ConsensusGroupId groupId, List<Peer> peers) throws Con
   @Override
   public void transferLeader(ConsensusGroupId groupId, Peer newLeader) throws ConsensusException {
     // first fetch the newest information
+    logger.info("transfer consensusGroup {} leader to {}", groupId, newLeader);
     final RaftGroupId raftGroupId = Utils.fromConsensusGroupIdToRaftGroupId(groupId);
     final RaftGroup raftGroup =
         Optional.ofNullable(getGroupInfo(raftGroupId))
             .orElseThrow(() -> new ConsensusGroupNotExistException(groupId));
-
+    if (raftGroup.getPeers() == null) {
```

Comment:
if we forbid leader trasnfer during regionMigration, thus we do not need this judgement?

Reply:
This commit is draft version, I'll delete these code in future commit

Reply:
OK, waiting for yuheng's pr

Reply:
remove it

Reply:
ok